wd_capabilities.au3

```diff
@@ -372,9 +372,6 @@ EndFunc   ;==>_WD_CapabilitiesNew
 Func __WD_CapabilitiesInitialize($s_MatchType, $s_BrowserName = '')
 	Local Const $sFuncName = "__WD_CapabilitiesInitialize"
 	#Region - parameters validation
-	If Not StringRegExp($s_MatchType, $_WD_CAPS_MATCHTYPE) Then _
```

Comment:
I remove this checking as it is already checked before calling `__WD_CapabilitiesInitialize()`

```autoit
If StringRegExp($key, $_WD_CAPS_MATCHTYPE) Then
......
```